src/common/components/accountBenefitsModal/accountBenefitsModal.tpl.html

```diff
@@ -0,0 +1,17 @@
+<div class="col-xs-12">
```

Comment:
I thought about re-use, but the html was small enough and functionality differed enough to keep them separate.